File: dsa/array_string_obj/fakeHeading.js
const action=["hello","hi","bye","Hey"]
const subject=["sam","mik","rik","sougata"]

function rand(arr){
    let index=Math.floor(Math.random()*arr.length)
    return arr[index];
}
function heading(){
    let first=rand(action);
    let second=rand(subject);
    let heading=`${first} - ${second}`;
    console.log(heading)
}
heading()

File: dsa/array_string_obj/fakeHeading.py
import random

subjects = [
    "Sougata Mukherjee",
    "Rupai",
    "Sam Muk",
    "a group of girls",
    "company CEO",
    "Bus driver in Bangalore"
]

actions = [
    "cancels",
    "swimming with",
    "sleeps",
    "celebrates"
]

def rand(arr):
    return random.choice(arr)

def fake_heading():
    while True:
        subject = rand(subjects)
        action = rand(actions)
        print(f"\nBreaking News: {subject} {action}!")
        user_input = input("Do you want another headline? (yes/no): ").strip().lower()
        if user_input in ("no", "n"):
            print("\nThanks for using Fake Headline Generator!")
            break

fake_heading()
